aws/resource_aws_waf_web_acl.go

```diff
@@ -17,6 +17,9 @@ func resourceAwsWafWebAcl() *schema.Resource {
 		Read:   resourceAwsWafWebAclRead,
 		Update: resourceAwsWafWebAclUpdate,
 		Delete: resourceAwsWafWebAclDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
Can you please add the `## Import` documentation to `website/docs/r/waf_web_acl.html.markdown`? Thanks! It'd probably also be good to note the rules limitation (for now).